src/index.js

```diff
@@ -1,13 +1,30 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './styles/index.scss';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
 import App from './components/App';
 import * as serviceWorker from './serviceWorker';
+import rootReducer from './reducers/index';
+
+const initialState = {
+  books: [
+    { id: Math.round(100 * Math.random()), title: 'Life of Action', category: 'Action' },
```

Comment:
[optional]
There is a package (uuid)[https://www.npmjs.com/package/uuid]. That generates ids that look like book ISBN numbers.